packages/charts/react-charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -265,16 +259,41 @@ export class DonutChartBase extends React.Component<IDonutChartProps, IDonutChar
         focusZonePropsInHoverCard={this.props.focusZonePropsForLegendsInHoverCard}
         overflowText={this.props.legendsOverflowText}
         {...this.props.legendProps}
+        // eslint-disable-next-line react/jsx-no-bind
+        onChange={this._onLegendSelectionChange.bind(this)}
```

Comment:
>this._onLegendSelectionChange.bind(this) [](http://example.com/codeflow?start=18&length=40)

do you need to use this twice?

Reply:
This is used just once

Reply:
yes, if we don't use .bind(this), tests are failing